### PR TITLE
WINDUP-2464: tech tags missing classifications

### PR DESCRIPTION
**Pull request by m-brophy**
https://issues.jboss.org/browse/WINDUP-2464

### CODE REVIEW

File: rules-reviewed/technology-usage/non-xml-rules-technology-usage.rhamt.xml [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+            </when>
+            <perform>
+                <technology-identified name="EJB XML">
+                    <tag name="Connect"/>
+                    <tag name="Bean"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>
+        <!--rule id="non-xml-technology-usage-03000">
```
Comment (jonathanvila) [inline rules-reviewed/technology-usage/non-xml-rules-technology-usage.rhamt.xml]: Why is this commented ? is not being used ? I see Enterprise Web Services in the spreadsheet

File: rules-reviewed/technology-usage/non-xml-rules-technology-usage.rhamt.xml [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+            </when>
+            <perform>
+                <technology-identified name="Hibernate Mapping">
+                    <tag name="Store"/>
+                    <tag name="Embedded"/>
+                    <tag name="Object Mapping"/>
+                </technology-identified>
+            </perform>
+        </rule>
+        <!--rule id="non-xml-technology-usage-09000">
```
Comment (jonathanvila) [inline rules-reviewed/technology-usage/non-xml-rules-technology-usage.rhamt.xml]: Why are these rules commented ?

File: rules-reviewed/technology-usage/non-xml-rules-technology-usage.rhamt.xml [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+            </when>
+            <perform>
+                <technology-identified name="JDBC (embedded)">
+                    <tag name="Store"/>
+                    <tag name="Database"/>
+                    <tag name="Java EE"/>
+                </technology-identified>
+            </perform>
+        </rule>
+        <!--rule id="non-xml-technology-usage-15000">
```
Comment (jonathanvila) [inline rules-reviewed/technology-usage/non-xml-rules-technology-usage.rhamt.xml]: why are these rules commented ?

File: rules-reviewed/technology-usage/tests/data/non-xml/batch.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<batch-artifacts xmlns="http://xmlns.jcp.org/xml/ns/javaee">
```
Comment (jonathanvila) [inline rules-reviewed/technology-usage/tests/data/non-xml/batch.xml]: As you modified the rule to consider this kind of xml files and use {*} , should the test follow that approach and then rename this file to : whatever.xml ? In order to see that the modification is working.

File: rules-reviewed/technology-usage/tests/non-xml-rules-technology-usage.rhamt.test.xml [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+          id="non-xml-technology-usage-test"
+          xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <testDataPath>data/non-xml/</testDataPath>
+    <rulePath>../../eap7/embedded-framework-libraries.windup.xml</rulePath>
+    <rulePath>../javaee-technology-usage.windup.xml</rulePath>
+    <rulePath>../non-xml-rules-technology-usage.rhamt.xml</rulePath>
+    <ruleset>
+        <rules>
+            <!--rule id="non-xml-technology-usage-01000-test">
```
Comment (jonathanvila) [inline rules-reviewed/technology-usage/tests/non-xml-rules-technology-usage.rhamt.test.xml]: why is this rule commented ?

File: rules-reviewed/technology-usage/tests/non-xml-rules-technology-usage.rhamt.test.xml [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+                            <tag name="Bean"/>
+                            <tag name="Java EE"/>
+                        </technology-statistics-exists>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Expected data not found for rule non-xml-technology-usage-02000"/>
+                </perform>
+            </rule>
+            <!--rule id="non-xml-technology-usage-03000-test">
```
Comment (jonathanvila) [inline rules-reviewed/technology-usage/tests/non-xml-rules-technology-usage.rhamt.test.xml]: why is this rule commented ?

File: rules-reviewed/technology-usage/tests/non-xml-rules-technology-usage.rhamt.test.xml [hunk 7]
```diff
@@ -0,0 +1,353 @@
...
+                            <tag name="Embedded"/>
+                            <tag name="Object Mapping"/>
+                        </technology-statistics-exists>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Expected data not found for rule non-xml-technology-usage-06000"/>
+                </perform>
+            </rule>
+            <!--rule id="non-xml-technology-usage-09000-test">
```
Comment (jonathanvila) [inline rules-reviewed/technology-usage/tests/non-xml-rules-technology-usage.rhamt.test.xml]: why are these rules commented ?

File: rules-reviewed/technology-usage/tests/non-xml-rules-technology-usage.rhamt.test.xml [hunk 8]
```diff
@@ -0,0 +1,353 @@
...
+                            <tag name="Database"/>
+                            <tag name="Java EE"/>
+                        </technology-statistics-exists>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="Expected data not found for rule non-xml-technology-usage-14000"/>
+                </perform>
+            </rule>
+            <!--rule id="non-xml-technology-usage-15000-test">
```
Comment (jonathanvila) [inline rules-reviewed/technology-usage/tests/non-xml-rules-technology-usage.rhamt.test.xml]: why are these rules commented ?

File: rules-reviewed/technology-usage/non-xml-rules-technology-usage.rhamt.xml (hunk 1 above)
Comment (m-brophy) [reply rules-reviewed/technology-usage/non-xml-rules-technology-usage.rhamt.xml]: I can't find anywhere where this tech tag is produced so although it was placed in the spreadsheet I've left the rule and test commented out. The other commented rules/tests are the same. We need to decide to remove them, first I think I need to discover why they are in the spreadsheet to ensure nothing has been missed.

File: rules-reviewed/technology-usage/tests/data/non-xml/batch.xml (hunk 4 above)
Comment (m-brophy) [reply rules-reviewed/technology-usage/tests/data/non-xml/batch.xml]: I've changed it back to batch.xml -  this was altered when I was struggling to get the rule to fire, but I forgot to replace the filename once I got it working.

**Pull request merged**